quilt_lite/quilt_lite/lambdas.py

```diff
@@ -0,0 +1,56 @@
+from base64 import b64decode, b64encode
+import importlib
+import sys
+
+from fastapi import FastAPI, HTTPException, Request, Response
+
+sys.path.extend([
+    '../lambdas/shared/',
+    '../lambdas/',
+])
```

Comment:
I think we should make it work without modifying `sys.path`.

Reply:
we definitely should. any ideas how?

Reply:
Symlink?